main_window/mapping/mapbox_utils.py

```diff
@@ -9,11 +9,11 @@
 import numpy as np
 from matplotlib import pyplot as plt
```

Comment:
```suggestion
if getattr(sys, 'frozen', False) and platform == 'darwin':
    matplotlib.use('Qt5Agg')
from matplotlib import pyplot as plt
```
Can also be the TkAgg backend